dashboard/src/components/DeploymentForm/DeploymentFormBody/DeploymentFormBody.tsx

```diff
@@ -0,0 +1,326 @@
+// Copyright 2019-2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CdsButton } from "@cds/react/button";
+import { CdsControlMessage } from "@cds/react/forms";
+import { CdsIcon } from "@cds/react/icon";
+import ConfirmDialog from "components/ConfirmDialog";
+import Alert from "components/js/Alert";
+import LoadingWrapper from "components/LoadingWrapper";
+import Tabs from "components/Tabs";
+import { isEmpty } from "lodash";
+import { FormEvent, RefObject, useCallback, useEffect, useState } from "react";
+import { retrieveBasicFormParams, updateCurrentConfigByKey } from "shared/schema";
+import { DeploymentEvent, IBasicFormParam, IPackageState } from "shared/types";
+import { getValueFromEvent } from "shared/utils";
+import { parseToYamlNode, setPathValueInYamlNode, toStringYamlNode } from "shared/yamlUtils";
+import YAML from "yaml";
+import AdvancedDeploymentForm from "./AdvancedDeploymentForm";
+import BasicDeploymentForm from "./BasicDeploymentForm/BasicDeploymentForm";
+
+export interface IDeploymentFormBodyProps {
+  deploymentEvent: DeploymentEvent;
+  packageId: string;
+  packageVersion: string;
+  deployedValues?: string;
+  packagesIsFetching: boolean;
+  selected: IPackageState["selected"];
+  appValues: string;
+  setValues: (values: string) => void;
+  setValuesModified: () => void;
+  formRef: RefObject<HTMLFormElement>;
+}
+
+function DeploymentFormBody({
+  deploymentEvent,
+  packageId,
+  packageVersion,
+  deployedValues: valuesFromTheDeployedPackage,
+  packagesIsFetching,
+  selected,
+  appValues: valuesFromTheParentContainer,
+  setValues: setValuesFromTheParentContainer,
+  setValuesModified,
+  formRef,
+}: IDeploymentFormBodyProps) {
+  const {
+    availablePackageDetail,
+    versions,
+    schema: schemaFromTheAvailablePackage,
+    values: valuesFromTheAvailablePackage,
```

Comment:
I've renamed the params to make it clearer what they really mean while developing. Happy to revert the change back, though.